hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/common/TaskMonitorTmpl.jamon

```diff
@@ -116,6 +117,7 @@ String parent = "";
                       <td><% task.getStatus() %>
                           (since <% StringUtils.formatTimeDiff(now, task.getStatusTime()) %>
                           ago)</td>
+                      <td><% task.getCompletionTimestamp() < 0 ? task.getState() : new Date(task.getCompletionTimestamp()) %></td>
```

Comment:
Do you want to add duration?

Reply:
I think that adding completion time is enough, there is already a start time. Do we still need duration?

Reply:
I think duration would be a nice-to-have. Can add in a follow-on. This is already a nice improvement as is. Thanks.